tests/modules/programs/qutebrowser/default.nix

```diff
@@ -2,4 +2,5 @@
   qutebrowser-settings = ./settings.nix;
   qutebrowser-keybindings = ./keybindings.nix;
   qutebrowser-quickmarks = ./quickmarks.nix;
+  qutebrowser-greasemonkey = ./greasemonkey.nix;
```

Comment:
these lines really should be sorted, but I guess it was already unsorted, so this is fine